ci/test

```diff
@@ -30,11 +30,26 @@ $BOSH deployments
 header "Cleaning up from any previous deployments (if necessary)..."
 cleanup ci-baseline-prometheus
 safe rm -rf secret/genesis-ci/prometheus
-# ci-baseline doesn't rely on external secrets
-# safe cp -rf secret/genesis-ci/static/prometheus secret/genesis-ci/prometheus/ci/baseline
 
 header "Deploying BASELINE environment to verify functionality..."
-genesis secrets --vault da-vault add ci-baseline
-genesis deploy -n ci-baseline
+genesis add-secrets ci-baseline --vault da-vault
+cat ci-baseline.yml
+pwd
+ls
+cat dev/kit.yml
+genesis -v
+genesis deploy -y ci-baseline
 $BOSH -d ci-baseline-prometheus instances --ps
+header "Validating BASELINE prometheus..."
+bosh -d ci-baseline-prometheus vms
+response=$(curl -k https://10.200.190.135:8080/-/healthy)
+if [[ $response == "Prometheus is Healthy." ]]; then
+  cleanup ci-baseline-prometheus
+  exit 0
+fi
+echo "Expected https://10.200.190.135:8080/-/healthy to return 'Prometheus is Health.' but instead" \
```

Comment:
Health? or Healthy?

Reply:
`/-/healthy` is correct. It checks to see if the webHandler is running (though does not check for other underlying Prometheus errors). There are no smoke-tests for Prometheus, so unfortunately this is the best we have. There are plans to improve `healthy` to check scrapers, etc. in the near future.

Reply:
The error message states:

> ... to return 'Prometheus is Health.' but ...

I think you forgot a charactr